qt/app.py

```diff
@@ -706,13 +706,29 @@ def _create_main_toolbar(self):
             self.act_shutdown,
         ]
 
-        toolbar.addActions(actions_for_toolbar)
+        # Add each action to toolbar
+        for act in actions_for_toolbar:
+            toolbar.addAction(act)
+
+            # Assume an explicit tooltip if it is different from "text()".
+            # Note that Qt use "text()" as "toolTip()" by default.
+            if act.toolTip() != act.text():
+
+                if toolbar.layoutDirection() == 1:
```

Comment:
I would prefer to use the enum constant `Qt.LayoutDirection.RightToLeft` instead of the magic number "1"...